dataflow/pkg/scheduler/lifecycle/lifecycle_sched.go

```diff
@@ -33,14 +28,18 @@ import (
 	datamover "github.com/opensds/multi-cloud/datamover/proto"
 	s3error "github.com/opensds/multi-cloud/s3/error"
 	s3utils "github.com/opensds/multi-cloud/s3/pkg/utils"
-	s3 "github.com/opensds/multi-cloud/s3/proto"
 	osdss3 "github.com/opensds/multi-cloud/s3/proto"
+	s3 "github.com/opensds/multi-cloud/s3/proto"
 	log "github.com/sirupsen/logrus"
 	"golang.org/x/net/context"
+	"os"
+	"sort"
+	"strconv"
+	"sync"
+	"time"
```

Comment:
as per convention, core packages should be on top of file

Reply:
even after go fmt, its still in bottom.

Reply:
Please refer : https://github.com/golang/go/wiki/CodeReviewComments#imports